marklogic-data-hub-central/ui/e2e/cypress/e2e/monitor/monitorTableValidations.cy.tsx

```diff
@@ -408,22 +408,15 @@ describe("Monitor Tile", () => {
   });
 
   it("Verify facets can be selected, applied and cleared using clear text", () => {
-    monitorPage.getFacetCheckbox("step-type", "ingestion").scrollIntoView().click({force: true});
+    monitorPage.clickFacetCheckbox("step-type", "ingestion");
     cy.waitForAsyncRequest();
```

Comment:
Should we remove this wait since we wait inside the clickFacetCheckbox?

Reply:
Yes, we should remove it.

Done!